docker-app/qfieldcloud/core/models.py

```diff
@@ -1083,6 +1083,17 @@ class PackagingOffliner(models.TextChoices):
         QGISCORE = "qgiscore", _("QGIS Core Offline Editing (deprecated)")
         PYTHONMINI = "pythonmini", _("Optimized Packager")
 
+    def get_localized_layers(self) -> list[dict]:
```

Comment:
```suggestion
    def get_localized_layers(self) -> list[LocalizedLayerDict]:
```
Can you please define a `LocalizedLayerDict(TypedDict)`, where you describe the fields with `total=False`?

Reply:
```suggestion
    @property
    def localized_layers(self) -> list[dict]:
```
Another proposal would be make the method a dynamic property?